### DISCUSSION TITLE
打包部署后点击左侧菜单请求文件js时，request headers 没有携带referer

### DISCUSSION THREAD

**Discussion by kk305cs** [vuepress-theme-hope/vuepress-theme-hope#2638] [Q&A] [answered]
由于项目服务有防盗链配置，打包部署后，点击左侧菜单时，network 请求文件 js 时，request headers 没有携带 referer，点击左侧菜单会打不开对应链接，并且 console 内报错 403 阻止访问该 js 文件
![image](https://user-images.githubusercontent.com/26682400/210916886-d21963d1-27fc-4d65-acfc-77ba34437602.png)

官网内也有 js 未携带 referer，这个问题是否有解决办法呢？
![Snipaste_2023-01-06_10-29-36](https://user-images.githubusercontent.com/26682400/210917724-28a92f35-992d-4620-bb35-0575f7c94828.png)

![image](https://user-images.githubusercontent.com/26682400/210917509-a2fa7ed4-a1cc-445d-bfeb-ea4abd7ae3d4.png)

**Comment by Mister-Hope** [comment, marked as answer]
Not familiar with this, probably an upstream issue, maybe you can ask this question in vuepress repo

**Reply by kk305cs** [reply]
ok

**Reply by kk305cs** [reply]
已找到问题所在，如果两个以上的 md 文件引入同一张图片就会出现此问题，需保证图片的唯一性，我处理的办法是将图片复制后重命名，即可，若大家遇到此问题可参考此方法解决。
